l10n/template.ftl

```diff
@@ -0,0 +1,288 @@
+not-found-back = Go back to the home page
+reference-toc-header = In this article
+footer-mofo = Visit <a data-l10n-name="moco">Mozilla Corporation’s</a> not-for-profit parent, the <a data-l10n-name="mofo">Mozilla Foundation</a>.
+footer-copyright = Portions of this content are ©1998–{ $year } by individual mozilla.org contributors. Content available under <a data-l10n-name="cc">a Creative Commons license</a>.
+search-modal-site-search = Site search for <em>{ $query }</em>
+site-search-search-stats = Found { $results } documents.
+site-search-suggestion-matches =
+    { $relation ->
+        [gt]
+            more than { $matches ->
+                [one] { $matches } match
+               *[other] { $matches } matches
+            }
+       *[eq]
+            { $matches ->
+                [one] { $matches } match
+               *[other] { $matches } matches
+            }
+    }
+site-search-suggestions-text = Did you mean:
+blog-time-to-read =
+    { $minutes ->
+        [one] { $minutes } minute read
+       *[other] { $minutes } minutes read
+    }
+blog-post-not-found = Blog post not found.
+blog-previous = Previous post
+blog-next = Next post
+-brand-name-obs = HTTP Observatory
+obs-report = Report
+obs-title = { -brand-name-obs }
+obs-landing-intro = Launched in 2016, the { -brand-name-obs } enhances web security by analyzing compliance with best security practices. It has provided insights to over 6.9 million websites through 47 million scans.
+obs-assessment = Developed by Mozilla, the { -brand-name-obs } performs an in-depth assessment of a site’s HTTP headers and other key security configurations.
+obs-scanning = Its automated scanning process provides developers and website administrators with detailed, actionable feedback, focusing on identifying and addressing potential security vulnerabilities.
+obs-security = The tool is instrumental in helping developers and website administrators strengthen their sites against common security threats in a constantly advancing digital environment.
+obs-mdn = The { -brand-name-obs } provides effective security insights, guided by Mozilla's expertise and commitment to a safer and more secure internet and based on well-established trends and guidelines.
+compat-loading = Loading…
+compat-browser-version-date = { $browser } { $version } – Released { $date }
+compat-browser-version-released = Released { $date }
+compat-link-report-issue = Report problems with this compatibility data
+compat-link-report-issue-title = Report an issue with this compatibility data
+compat-link-report-missing-title = Report missing compatibility data
+compat-link-report-missing = Report this issue
+compat-link-source = View data on GitHub
+compat-link-source-title = File: { $filename }
+compat-deprecated = Deprecated
+compat-experimental = Experimental
+compat-nonstandard = Non-standard
+compat-no = No
+compat-support-full = Full support
```

Comment:
I'm finding this and multiple other strings with the exact same ID. You'll need to find a way to avoid duplication.